src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -76,6 +76,8 @@ public void setInternalState(InternalState internalState) {
 
     @Override
     public InternalState getInternalState() {
+        // Update before returning internal state
+        internalState.updateInternalState();
```

Comment:
I think this is being done somewhere else, we should remove it, this is a better way yes.

Reply:
Ok I was wrong, nothing to replace.